tests/test_application.py

```diff
@@ -267,3 +268,45 @@ def helper(scopes, account, authority, *args, **kwargs):
         with self.assertRaises(ExpectedBehavior):
             self.app.acquire_token_silent(["different scope"], self.account)
 
+
+class TestApplicationForClientCapabilities(unittest.TestCase):
+
+    def test_capabilities_and_id_token_claims_merge(self):
+        client_capabilities = ["llt", "ssm"]
```

Comment:
Are `llt` and `ssm` mocked-up values? Per the [doc](https://microsoft.sharepoint.com/:w:/t/aad/protocols/ERSWYtOQB45GgG4e2q3Cz00B6C36zi4gAs6JhdQG_wvVeQ?rtime=yEUj_nvd1kg&wdLOR=cB0CF5D1C-739A-4A91-983A-DBBDB9E775A6):

> The currently known values are: CP1.

This may confuse the readers.

Reply:
Good point. Perhaps we should have changed them into "foo", "bar". @abhidnya13